pkg/controller/azdrivernode.go

```diff
@@ -83,6 +85,9 @@ func (r *ReconcileAzDriverNode) Reconcile(ctx context.Context, request reconcile
 
 // run an update on existing azdrivernode objects to store them under new version if necessary
 func (r *ReconcileAzDriverNode) Recover(ctx context.Context) error {
+	r.SharedState.nodeDiskAvailabilityMap.Range(func(k, v interface{}) bool {
+		return true
+	})
```

Comment:
What's the purpose of this? It appears to be a no-op since the range function does nothing to the elements of the map.

Reply:
Deleted